sample/src/main/java/org/zalando/failsafeactuator/sample/SampleApplicationBanner.java

```diff
@@ -1,27 +0,0 @@
-package org.zalando.failsafeactuator.sample;
```

Comment:
Can you recover the Banner? I actually liked that.

Reply:
Yes please :)